### PR TITLE
Benchmarks components: plotting utils

### PR DESCRIPTION
**Pull request by yger**
Commits for the plotting function of the sorting components, that have been made to prepare the talk of Edinburgh. Worth considering all of them, but WIP to refactor more the plots before a paper

### CODE REVIEW

**Review by samuelgarcia — APPROVED** [review]

**Comment by samuelgarcia** [comment]
@alejoe91  : I will merge this squash if this is OK for you. This is only components stuff.

**Comment by alejoe91** [comment]
Ok!

File: src/spikeinterface/comparison/groundtruthstudy.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 from spikeinterface.core import load_extractor, create_sorting_analyzer, load_sorting_analyzer
 from spikeinterface.core.core_tools import SIJsonEncoder
 from spikeinterface.core.job_tools import split_job_kwargs
+from spikeinterface.widgets import get_some_colors
```
[the remark this answers is not in the document]
Comment (yger) [reply src/spikeinterface/comparison/groundtruthstudy.py]: done

**Comment by yger** [comment]
Fine for squashing and merging

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**